src/display/transport_stream.js

```diff
@@ -0,0 +1,241 @@
+/* Copyright 2012 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assert, createPromiseCapability } from '../shared/util';
+
+ /** @implements {IPDFStream} */
+var PDFDataTransportStream = (function PDFDataTransportStreamClosure() {
+  function PDFDataTransportStream(params, pdfDataRangeTransport) {
+    this.transportReadyCapability = createPromiseCapability();
```

Comment:
Actually I am using `this.transportReadyCapability` to signal `this._pdfDataRangeTransport.transportReady()` [here](https://github.com/mozilla/pdf.js/pull/8617/commits/e18b71d617a561e819087ed334c52884de56402e#diff-3089181c2189d3ef27f3dbd49ca08965R50), `this.transportReadyCapability` is resolved only when we get [PDFManagerReady](https://github.com/mozilla/pdf.js/pull/8617/commits/e18b71d617a561e819087ed334c52884de56402e#diff-0ecad279ed2252e3eb47a4d96ec1463cL1731) message from worker.

I am doing something wrong here?